hack/jenkins/update-jobs.sh

```diff
@@ -42,4 +42,4 @@ fi
 
 docker exec job-builder git checkout master
 docker exec job-builder git pull
-docker exec job-builder jenkins-jobs test ${config_dir}
+docker exec job-builder jenkins-jobs update ${config_dir}
```

Comment:
I wonder if that plugin shows the diff or only notified that something changed. The former is far more useful than the latter...

In any case, seems like we should install that plugin and then update the configs here to set notifications onUpdate and onDelete?